api/api/utils/photon.py

```diff
@@ -99,6 +139,19 @@ def get(
         raise UpstreamThumbnailException(
             f"Failed to render thumbnail due to timeout: {exc}"
         )
+    except requests.HTTPError as exc:
+        # Save occurrences to redis and only sent to Sentry when reached a
+        # threshold to prevent providers downtime consuming all the events
+        # quota.
```

Comment:
```suggestion
        # Save number of occurrences to redis. Only raise a Sentry error when the number of
        # occurrences reaches the configured threshold, to prevent provider API downtime from
        # causing excessive Sentry events.
```